src/icrar/leap-accelerate-client/Calibrate.cc

```diff
@@ -64,7 +64,7 @@ namespace icrar
         MeasurementSet ms = MeasurementSet(ms_filename);
         throw std::runtime_error("not implemented");
 
-        MetaData metadata = MetaData(ms);
+        casalib::MetaData metadata = casalib::MetaData(ms);
```

Comment:
I put the functions using casacore matrices in the `icrar::casalib` namespace since the old casacore version has the `casa -> casacore` macro that often caused problems on xenial if I otherwise named it `icrar::casa`.

Reply:
another possible name for casalib is `ported`